go.mod

```diff
@@ -3,18 +3,21 @@ module k8s.io/cloud-provider-vsphere
 go 1.12
 
 require (
+	github.com/davecgh/go-xdr v0.0.0-20161123171359-e6a2ba005892 // indirect
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
 	github.com/docker/distribution v2.7.1+incompatible // indirect
 	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
 	github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6 // indirect
 	github.com/golang/protobuf v1.3.2
 	github.com/google/btree v1.0.0 // indirect
 	github.com/imdario/mergo v0.3.7 // indirect
+	github.com/kr/pretty v0.1.0 // indirect
 	github.com/opencontainers/go-digest v1.0.0-rc1 // indirect
 	github.com/prometheus/client_golang v0.9.3-0.20190127221311-3c4408c8b829
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/pflag v1.0.3
-	github.com/vmware/govmomi v0.21.0
+	github.com/vmware/govmomi v0.20.2
```

Comment:
any reason to go back to `v0.20.2` from `v0.21.0`?

Reply:
Whoops, I had bumped from `v0.20.0` to `v0.20.2` before rebasing against your modules update, I'll revert this.

Reply:
👍